sched/group/group_addrenv.c

```diff
@@ -98,11 +161,11 @@ int group_addrenv(FAR struct tcb_s *tcb)
     }
 
   DEBUGASSERT(tcb && tcb->group);
```

Comment:
This DEBUGASSERT() still causes a crash as tcb->group can be NULL here. I will do the fix tomorrow.